src/include/planner/plannodes/create_trigger_plan_node.h

```diff
@@ -138,8 +138,8 @@ class CreateTriggerPlanNode : public AbstractPlanNode {
      * Build the create trigger plan node
      * @return plan node
      */
-    std::shared_ptr<CreateTriggerPlanNode> Build() {
-      return std::shared_ptr<CreateTriggerPlanNode>(new CreateTriggerPlanNode(
+    std::unique_ptr<CreateTriggerPlanNode> Build() {
```

Comment:
I am kind of confuse. What is the purpose of Build and how is it different from `CreateTriggerPlanNode` . Why not expose the constructor?

Reply:
Not particularly familiar with this usage.
@wenxuanqiu Opinions?